WIP: Fix translations and udpdate documentation

3d-gussner opened the pull request:
- [ ] Fix missing/duplicate translations
  - [ ] Czech
  - [ ] Spanish
  - [ ] French
  - [ ] Italian
  - [ ] Polish
- [ ] Update translation po files
- [X] Update RepRap Wiki
- [ ] Review Readme.md
- [ ] Review Add_new_language.md
- [ ] Review translation.md

awenelo commented on lang/po/Firmware_fr.po:
```diff
@@ -601,87 +611,92 @@ msgid "Checking Y axis"
+# MSG_WIZARD_WELCOME_SHIPPING c=20 r=16
+#: messages.c:122
+msgid "Hi, I am your Original Prusa i3 printer. I will guide you through a short setup process, in which the Z-axis will be calibrated. Then, you will be ready to print."
+msgstr "Bonjour, je suis votre imprimante Original Prusa i3. Je vais vous accompagner au cours d'un bref processus de reglage, qui permettra de calibrer le Z-axis. Apres cela, tout sera pret pour imprimer."
```
```suggestion
msgstr "Bonjour, je suis votre imprimante Original Prusa i3. Je vais vous accompagner au cours d'un processus bref de reglage pour calibrer le Z-axis. Apres cela, tout sera pret pour imprimer."
```

##### `bref processus` -> `processus bref`
`bref` is describing the length, not the size of the process, and should come after the noun.

##### `processus de relage, qui permettra de calibrer` -> `processus de relage pour calibrer`
`qui permettra de calibrer` sounds like you're saying `One of the things you can do in this calibration process is calibrate the Z-axis, but it's not necessary for printing`. If the process is to calibrate the Z-axis and it's necessary to print, then you should say `pour calibrer`.

3d-gussner committed:
Update "Crash detected. Resume print?" c=20 r=2

3d-gussner committed:
Update "Now remove the test print from steel sheet." c=20 r=4

3d-gussner committed:
Update #MSG_FIND_BED_OFFSET_AND_SKEW_LINE1 c=20 r=2

3d-gussner committed:
Fix too long translations in MSG_USERWAIT c=20

3d-gussner committed:
Update MSG_WAITING_TEMP c=20 r=4

3d-gussner committed:
Update #MSG_GCODE_DIFF_PRINTER_CANCELLED c=20 r=8

3d-gussner committed:
Update "Printer nozzle diameter differs from the G-code. Continue?" c=20 r=5

3d-gussner committed:
Update "Printer nozzle diameter differs from the G-code. Please check the value in settings. Print cancelled." c=20 r=9
as French translation is longer

3d-gussner committed:
Update "If you have additional steel sheets, calibrate their presets in Settings - HW Setup - Steel sheets." c=20 r=9
as German translation is longer

3d-gussner committed:
Update "Please insert filament into the extruder, then press the knob to load it." c=20 r=6
as German translation is longer

3d-gussner committed:
Fix too long German translations

3d-gussner committed:
Update "Select nozzle preheat temperature which matches your material." c=20 r=6
as Spanish translation is longer

3d-gussner committed:
Fix too long Spanish translation

3d-gussner committed:
Update "Choose a filament for the First Layer Calibration and select it in the on-screen menu."
as French translation is longer

3d-gussner committed:
Update "G-code sliced for a different level. Please re-slice the model again. Print cancelled."
as French translation is longer

3d-gussner committed:
Update "G-code sliced for a newer firmware. Continue?"
as French translation is longer

3d-gussner committed:
Update "G-code sliced for a newer firmware. Please update the firmware. Print cancelled."
as French translation is longer

3d-gussner committed:
Update MSG_NEW_FIRMWARE_AVAILABLE c=20 r=2

3d-gussner committed:
Update #MSG_DEFAULT_SETTINGS_LOADED c=20 r=6
as Italian translation is longer

3d-gussner committed:
Update MSG_PLACE_STEEL_SHEET c=20 r=5
as Italian translation is longer

3d-gussner committed:
Update MSG_CALIBRATE_BED_RESET c=18

3d-gussner committed:
Update MSG_BED_SKEW_OFFSET_DETECTION_FAILED_FRONT_BOTH_FAR c=20 r=6
as Polish translation is longer

3d-gussner committed:
Update MSG_BED_SKEW_OFFSET_DETECTION_FAILED_FRONT_RIGHT_FAR c=20 r=6
as Polish translation is longer

3d-gussner committed:
Fix typo "didnt" to "didn't"

3d-gussner committed:
Update MSG_BED_LEVELING_FAILED_POINT_LOW c=20 r=6
as Dutch translation is longer

3d-gussner committed:
Update MSG_NEW_FIRMWARE_PLEASE_UPGRADE c=20
Fix Dutch translation to fit in 20 chars

3d-gussner committed:
Fix too long Dutch translation

3d-gussner committed:
Fix some Dutch translations

3d-gussner committed:
Update MSG_MMU_POWER_FAILS c=15

3d-gussner committed:
Update MSG_MEASURED_OFFSET c=20

3d-gussner committed:
Create MSG_CANCEL c=9
Delete ">Cancel"
Samed 10 bytes
Updated MSG_MEASURED_OFFSET c=20

3d-gussner committed:
Update MSG_AMBIENT c=14

3d-gussner committed:
Update MSG_AUTO_HOME c=18

3d-gussner committed:
Update MSG_SELFTEST_AXIS_LENGTH c=20

3d-gussner committed:
Update MSG_SELFTEST_AXIS c=16

3d-gussner committed:
Update MSG_SELFTEST_SWAPPED c=16

3d-gussner committed:
Update MSG_SELFTEST_BEDHEATER c=20

3d-gussner committed:
Update MSG_SELFTEST_HEATERTHERMISTOR c=20

3d-gussner committed:
Update MSG_SELFTEST_NOTCONNECTED c=20

3d-gussner committed:
Update MSG_SELFTEST_ERROR c=20

3d-gussner committed:
Update MSG_SELFTEST_PLEASECHECK c=20

3d-gussner committed:
Update MSG_SELFTEST_ENDSTOPS c=20

3d-gussner committed:
Update MSG_SELFTEST_ENDSTOP c=16

3d-gussner committed:
Update MSG_LOOSE_PULLEY c=20

3d-gussner committed:
Update MSG_BED_DONE c=20

3d-gussner committed:
Update MSG_BED_HEATING c=20

3d-gussner committed:
Update #MSG_BED_CORRECTION_MENU c=18

3d-gussner committed:
Update #MSG_BED c=13

3d-gussner committed:
Update #MSG_CALIBRATE_BED c=18

3d-gussner committed:
Update #MSG_HOMEYZ c=18

3d-gussner committed:
Update #MSG_HOMEYZ_DONE c=20

3d-gussner committed:
Update #MSG_MENU_CALIBRATION c=18

3d-gussner committed:
Update #MSG_SD_REMOVED c=20

3d-gussner committed:
Define MSG_CHECKING_FILE c=17

3d-gussner committed:
Update #MSG_NOT_COLOR c=19

3d-gussner committed:
Update #MSG_COOLDOWN c=18

3d-gussner committed:
Define #MSG_COPY_SEL_LANG c=20 r=3

3d-gussner committed:
Define #MSG_CALIBRATING_HOME c=20

3d-gussner committed:
Define #MSG_CHOOSE_FIL_1ST_LAYERCAL c=20 r=7

3d-gussner committed:
Define #MSG_CRASH_RESUME c=20 r=3

3d-gussner committed:
Update #MSG_DISABLE_STEPPERS c=18

3d-gussner committed:
Define #MSG_MMU_CUTTING_FIL c=18

3d-gussner committed:
Update #MSG_ERROR c=10

3d-gussner committed:
Define #MSG_MMU_FAIL_STATS c=18

3d-gussner committed:
Define #MSG_FAIL_STATS c=18

3d-gussner committed:
Update #MSG_FSENSOR c=12

3d-gussner committed:
Define #MSG_MMU_FIX_ISSUE c=20 r=4

3d-gussner committed:
Update #MSG_FLOW c=15

3d-gussner committed:
Update #MSG_PRUSA3D_FORUM c=18

3d-gussner committed:
Update #MSG_PRUSA3D c=18

3d-gussner committed:
Update #MSG_PRUSA3D_HOWTO c=18

3d-gussner committed:
Update #MSG_HEATING c=20

3d-gussner committed:
Update #MSG_FILAMENTCHANGE c=18

3d-gussner committed:
Update #MSG_CHANGE_SUCCESS c=20

3d-gussner committed:
Update #MSG_WATCH c=18

3d-gussner committed:
Define #MSG_ADDITIONAL_SHEETS c=20 r=9

3d-gussner committed:
Define #MSG_LEFT c=10

3d-gussner committed:
Define #MSG_RIGHT c=10

3d-gussner committed:
Update MSG_LANGUAGE_SELECT c=18

3d-gussner committed:
Update #MSG_SELFTEST_OK c=20

3d-gussner committed:
Update #MSG_SELFTEST c=18

3d-gussner committed:
Update #MSG_BELTTEST c=18

3d-gussner committed:
Define #MSG_SEL_PREHEAT_TEMP c=20 r=6

3d-gussner committed:
Update #MSG_SETTINGS c=18

3d-gussner committed:
Update #MSG_SORT c=7

3d-gussner committed:
Update #MSG_SORT_ALPHA c=8 #MSG_SORT_TIME c=8

3d-gussner committed:
Update #MSG_RPI_PORT c=13

3d-gussner committed:
Update #MSG_SOUND c=9

3d-gussner committed:
Update #MSG_SOUND_BLIND c=7, #MSG_SOUND_LOUD c=7, MSG_SOUND_ONCE c=7

3d-gussner committed:
Update #MSG_MAIN c=18

3d-gussner committed:
Update #MSG_NONE c=8

3d-gussner committed:
Update #MSG_WARN c=8

3d-gussner committed:
Update #MSG_STRICT c=8

3d-gussner committed:
Update #MSG_MODEL c=8

3d-gussner committed:
Update #MSG_FIRMWARE c=8

3d-gussner committed:
Update #MSG_NOZZLE_DIAMETER c=10, MSG_GCODE c=8

3d-gussner committed:
Update #MSG_MMU_MODE c=8

3d-gussner committed:
Update #MSG_SD_CARD c=8

3d-gussner committed:
Update #MSG_TOSHIBA_FLASH_AIR_COMPATIBILITY c=8

3d-gussner committed:
Update #MSG_MESH c=12

3d-gussner committed:
Update #MSG_Z_PROBE_NR c=14

3d-gussner committed:
Update #MSG_MAGNETS_COMP c=13

3d-gussner committed:
Update #MSG_CARD_MENU c=18

3d-gussner committed:
Update MSG_BACK c=18

3d-gussner committed:
Update #MSG_NOZZLE c=12

3d-gussner committed:
Update MSG_REFRESH c=18

3d-gussner committed:
Update MSG_RESUMING_PRINT c=20

3d-gussner committed:
Update #MSG_SELFTEST_MOTOR c=18

3d-gussner committed:
Update #MSG_SELFTEST_WIRINGERROR c=17

3d-gussner committed:
Update #MSG_MODE c=9

3d-gussner committed:
Update #MSG_MODE c=6

3d-gussner committed:
Update #MSG_HIGH_POWER c=10

3d-gussner committed:
Update #MSG_AUTO_POWER c=10

3d-gussner committed:
Update #MSG_STEALTH c=7, #MSG_SILENT c=7, #MSG_NORMAL c=7

3d-gussner committed:
Update #MSG_STOP_PRINT c=18

3d-gussner committed:
Update #MSG_STOPPED c=20

3d-gussner committed:
Update #MSG_NA c=3, #MSG_OFF c=3, #MSG_ON c=3

3d-gussner committed:
Update #MSG_AUTO_DEPLETE c=13

3d-gussner committed:
Update #MSG_NO c=3

3d-gussner committed:
Update #MSG_BABYSTEP_Z c=18

3d-gussner committed:
Update MSG_LOADING_COLOR c=20

3d-gussner committed:
Update #MSG_LOAD_TO_NOZZLE c=18

3d-gussner committed:
Update #MSG_MEASURE_BED_REFERENCE_HEIGHT_LINE1 c=20 r=3

3d-gussner committed:
Define #MSG_MEASURED_SKEW c=13

3d-gussner committed:
Define MSG_MMU_LOAD_FAILED c=20

3d-gussner committed:
Define #MSG_MMU_CONNECTED c=18

3d-gussner committed:
Update #MSG_MOVE_AXIS c=18

3d-gussner committed:
Update MSG_MOVE_X/Y/Z c=18

3d-gussner committed:
Update #MSG_NO_MOVE c=20

3d-gussner committed:
Update #MSG_YES c=3

3d-gussner committed:
Define #MSG_Y_DIST_FROM_MIN c=20

3d-gussner committed:
Update #MSG_FIL_FAILD c=20 r=5

3d-gussner committed:
Define #MSG_GCODE_DIFF_CONTINUE c=20 r=4, #MSG_GCODE_DIFF_CANCELLED c=20 r=7

3d-gussner committed:
Define #MSG_GCODE_NEWER_FIRMWARE_CONTINUE c=20 r=5, #MSG_GCODE_NEWER_FIRMWARE_CANCELLED c=20 r=8

3d-gussner committed:
Define #MSG_PREHEATING_TO_LOAD/UNLOAD/CUT/EJECT c=20

3d-gussner committed:
Define #MSG_TO_LOAD_FIL c=20, #MSG_TO_UNLOAD_FIL c=20, #MSG_PRESS_KNOB c=20

3d-gussner committed:
Define #MSG_FS_VERIFIED c=20 r=3

3d-gussner committed:
Define #MSG_LIN_CORRECTION c=18

3d-gussner committed:
Define #MSG_INSERT_FIL c=20 r=6

3d-gussner committed:
Define #MSG_REMOVE_TEST_PRINT c=20 r=4

3d-gussner committed:
Define #MSG_PINDA c=6 and remove duplicate saved 8 bytes

3d-gussner committed:
Fix #MSG_CRASHDETECT c=13 in Polish lang file

3d-gussner committed:
Update MSG_NO_CARD c=18

3d-gussner committed:
Define #MSG_NOZZLE_FAN c=10, #MSG_PRINT_FAN c=10

3d-gussner committed:
Define #MSG_REMOVE_SHIPPING_HELPERS c=20 r=3

3d-gussner committed:
Update MSG_PREHEAT c=18

3d-gussner committed:
Update #MSG_TUNE c=18, #MSG_PREHEAT c=18

3d-gussner committed:
Define #MSG_RESUME_NOZZLE_TEMP c=20 r=4

3d-gussner committed:
Define #MSG_MMU_INSERT_FILAMENT_FIRST_TUBE c=20 r=6

3d-gussner committed:
Define #MSG_UNLOAD_FILAMENT_REPEAT c=20 r=4

3d-gussner committed:
Define #MSG_CHECK_IR_CONNECTION c=20 r=4

3d-gussner committed:
Define #MSG_SLIGHT_SKEW c=13, #MSG_SEVERE_SKEW c=13

3d-gussner committed:
Update #MSG_SOUND_LOUD c=7

3d-gussner committed:
Define #MSG_RUNOUTS c=7

3d-gussner committed:
Define #MSG_Z-LEVELING_ENFORCED c=20 r=4

3d-gussner committed:
Update #MSG_SPEED c=15

3d-gussner committed:
Update #MSG_STATISTICS c=18

3d-gussner committed:
Update #MSG_SUPPORT c=18

3d-gussner committed:
Define #MSG_SELECT_FILAMENT c=20

3d-gussner committed:
Define #MSG_SELECT_TEMP_MATCHES_MATERIAL c=20 r=4

3d-gussner committed:
Update MSG_TEMPERATURE c=18

3d-gussner committed:
Define and update #MSG_PRINT_TIME c=19, #MSG_FILAMENT_USED c=19, #MSG_TOTAL_FILAMENT c=19, #MSG_TOTAL_PRINT_TIME c=19

3d-gussner committed:
Delete "Unload" as it isn't used anymore

3d-gussner committed:
Define #MSG_UNKNOWN c=13

3d-gussner committed:
Update #MSG_BACK c=18

3d-gussner committed:
Define #MSG_CHECKS c=18

3d-gussner committed:
Define #MSG_FALSE_TRIGGERING c=20

3d-gussner committed:
Define #MSG_FINDA c=5 and update #MSG_PINDA c=5

3d-gussner committed:
Define #MSG_MK3S_FIRMWARE_ON_MK3 c=20 r=4, #MSG_MK3_FIRMWARE_ON_MK3S c=20 r=4

3d-gussner committed:
Define #MSG_MODE_CHANGE_IN_PROGRESS c=20 r=3

3d-gussner committed:
Define #MSG_NOZZLE_DIFFERS_CONTINUE c=20 r=5, #MSG_NOZZLE_DIFFERS_CANCELLED c=20 r=9

3d-gussner committed:
Define #MSG_RENAME c=18

3d-gussner committed:
Define #MSG_SELECT c=18

3d-gussner committed:
Update #MSG_INFO_SENSORS c=18

3d-gussner committed:
Fix Dutch #MSG_FINDA c=5

3d-gussner committed:
Remove r=1 and some minor fixes

3d-gussner committed:
Cleanup some comments ////c=

3d-gussner committed:
Define #MSG_PRINTER_IP c=18

3d-gussner committed:
Fix output "Y distance from min" by removing ":" so translations can use 20 chars

3d-gussner committed:
Updated RepRap Wiki and removed the @todos

3d-gussner committed:
Update RepRap url for `M123`

3d-gussner commented:
@DRracer can you try to fix Czech translations?
https://github.com/3d-gussner/Prusa-Firmware/blob/PFW-1189/lang/lang_en_cz.txt#L141-L143
https://github.com/3d-gussner/Prusa-Firmware/blob/PFW-1189/lang/lang_en_cz.txt#L865-L867
```
count=364
offs=744
size=10202
lang=cz
Start cz lang-check
[E]: Text "Kalibruji vychozi poz." is longer then definiton on line 141 rows diff=1

[E]: Text "Stisknete hl. tlacitko" is longer then definiton on line 865 rows diff=1

End cz lang-check
```

3d-gussner commented:
Can anyone try to fix Spanish translations?
https://github.com/3d-gussner/Prusa-Firmware/blob/PFW-1189/lang/lang_en_es.txt#L141-L143
https://github.com/3d-gussner/Prusa-Firmware/blob/PFW-1189/lang/lang_en_es.txt#L313-L315
https://github.com/3d-gussner/Prusa-Firmware/blob/PFW-1189/lang/lang_en_es.txt#L321-L323
https://github.com/3d-gussner/Prusa-Firmware/blob/PFW-1189/lang/lang_en_es.txt#L405-L407
https://github.com/3d-gussner/Prusa-Firmware/blob/PFW-1189/lang/lang_en_es.txt#L537-L539
https://github.com/3d-gussner/Prusa-Firmware/blob/PFW-1189/lang/lang_en_es.txt#L641-L643
https://github.com/3d-gussner/Prusa-Firmware/blob/PFW-1189/lang/lang_en_es.txt#L1293-L1295

```
count=364
offs=744
size=11221
lang=es
Start es lang-check
[E]: Text "Calibrar posicion inicial" is longer then definiton on line 141 rows diff=1

[E]: Text "ERROR: El sensor de filamento no responde, por favor comprueba la conexion." is longer then definiton on line 289 rows diff=1

[E]: Text "Estadistica de fallos MMU" is longer then definiton on line 313 rows diff=1

[E]: Text "Estadistica de fallos" is longer then definiton on line 321 rows diff=1

[E]: Text "Ventiladores frontal/izquierdo" is longer then definiton on line 405 rows diff=1

[E]: Text "Correccion de Linealidad" is longer then definiton on line 537 rows diff=1

[E]: Text "Fallo de energia en MMU" is longer then definiton on line 641 rows diff=1

[E]: Text "Distancia en Y desde el min" is longer then definiton on line 1293 rows diff=1

End es lang-check

```

DRracer committed:
Merge pull request #3124 from prusa3d/MK3

Sync MK3 -> MK3_3.10.0

3d-gussner committed:
Merge remote-tracking branch 'upstream/MK3_3.10.0' into PFW-1189

3d-gussner committed:
Fix merge issues

3d-gussner committed:
Use textwarp to check the message length and rows
@DRracer: Added origial English text to the output, added cols and rows as well

3d-gussner committed:
Update "\04Refresh" message as two "abcd" "efgh" in one line cause issues in lang-build scripts

3d-gussner committed:
Readded `>Cancel` as the `>` has some meaning. Sadly the previously 10bytes more free space are gone again

3d-gussner committed:
Fix Typo in MSG_FIL_FAILED

3d-gussner committed:
Added ` EXTENDED_M20` to RepRap Gcode wiki `M115` see https://reprap.org/mediawiki/index.php?title=G-code&type=revision&diff=189034&oldid=189033

wavexx committed:
Allow to customize OUTDIR in config.sh

Useful for custom build trees.

wavexx committed:
Be more strict when checking translation lenght

Strip quotes immediately so that we can measure strings easily.
For single-line strings, check the translation lenght correctly.

wavexx committed:
Do not output colors when redirecting output

wavexx committed:
Remove warnings caused by '\x00'

Manually unescape '\x00' for the time being, removing spurious warnings.

Commit:
Fix length of some CZ messages

3d-gussner committed:
@leptun strip 1st char hack for duplicate `Cancel`

3d-gussner committed:
Merge branch 'PFW-1189' of https://github.com/3d-gussner/Prusa-Firmware into PFW-1189

3d-gussner committed:
Fix MSG_UNLOAD_FILAMENT length as in MMU a number is added

3d-gussner committed:
Fix Dutch +  German MSG_UNLOAD_FILAMENT too long translations

3d-gussner committed:
Fix German MSG_LOAD_TO_NOZZLE

3d-gussner committed:
Fix MSG_NO being too short, no need to be

3d-gussner committed:
Fix MSG_SELFTEST_WIRINGERROR to c=18

3d-gussner committed:
Fix German too long translation MSG_MMU_MODE

3d-gussner committed:
Fix typo thanks to @wavexx

wavexx committed:
lang-check: print a warning if cols!=20 on multiline messages

wavexx committed:
lang-check: simplify wrapping, wrap according to cols

wavexx committed:
lang-check: pretty-print errors to aid in translation

wavexx committed:
Update Italian translations

wavexx committed:
lang-check: highlight truncated strings

wavexx committed:
lang-check: handle backslash sequences

3d-gussner committed:
Fix Dutch FINDA translation

wavexx committed:
lang-check: respect --no-warning

wavexx committed:
lang-check: add basic punctuation checks

wavexx committed:
lang-check: attempt at spotting short translations

wavexx committed:
lang-check: cleanup

wavexx committed:
lang-check: add checks for % escapes

wavexx committed:
Two Italian translation improvements

wavexx committed:
lang-check: improve error strings

wavexx committed:
lang-check: handle/warn about trailing whitespace

wavexx committed:
Italian translation: remove trailing whitespace

3d-gussner committed:
Update German translation

3d-gussner committed:
Set MSG_SET_TEMPERATURE chars to c=20
Fix Italian warning shown by `lang-check.py`

3d-gussner committed:
Update lang/lang_en_fr.txt

Co-authored-by: awenelo <56133692+awenelo@users.noreply.github.com>

3d-gussner committed:
Update French translation for MSG_BED_CORRECTION_MENU by @carlin57

3d-gussner committed:
Update lang/lang_en_fr.txt

Co-authored-by: awenelo <56133692+awenelo@users.noreply.github.com>

3d-gussner committed:
Update lang/lang_en_fr.txt

Co-authored-by: awenelo <56133692+awenelo@users.noreply.github.com>

3d-gussner committed:
Update lang/lang_en_fr.txt

Co-authored-by: awenelo <56133692+awenelo@users.noreply.github.com>

3d-gussner committed:
Update lang/lang_en_fr.txt

Co-authored-by: awenelo <56133692+awenelo@users.noreply.github.com>

3d-gussner committed:
Merge branch 'PFW-1189' of https://github.com/3d-gussner/Prusa-Firmware into PFW-1189

3d-gussner committed:
Update French translation for MSG_SELECT_FILAMENT as @carlin57 suggested

3d-gussner committed:
Use origin message for MSG_TOSHIBA_FLASH_AIR_COMPATIBILITY also in French

3d-gussner committed:
Updated French translation of MSG_MMU_LOAD_FAILED c=20 as @awenelo suggested

3d-gussner committed:
Update lang/lang_en_fr.txt

Co-authored-by: awenelo <56133692+awenelo@users.noreply.github.com>

3d-gussner committed:
Update lang/lang_en_fr.txt

Co-authored-by: awenelo <56133692+awenelo@users.noreply.github.com>

3d-gussner committed:
Merge branch 'PFW-1189' of https://github.com/3d-gussner/Prusa-Firmware into PFW-1189

3d-gussner committed:
Fix French translation for MSG_MMU_LOAD_FAILS

3d-gussner committed:
Use same wording in Spanish translation for "knob"

3d-gussner committed:
Fix typo in Spanish

3d-gussner committed:
Remove white spaces in messags which saves ~62bytes

3d-gussner committed:
Revert that has been commited by mistake

Commit:
Fix some trailing whitespace and puctuation CZ

3d-gussner committed:
Remove white spaces in cz,nl,pl translations

3d-gussner committed:
Merge branch 'PFW-1189' of https://github.com/3d-gussner/Prusa-Firmware into PFW-1189

3d-gussner committed:
Remove white spaces and shortn some messages (inspired by FR, IT)

3d-gussner committed:
Add character ruler

3d-gussner committed:
Update MSG_SEVER_SKEW and MSG_SLIGHT_SKEW message to c=14
Changed and tested Support-> "XYZ cal. details" LCD output

3d-gussner committed:
Fix character ruler for too many rows

wavexx committed:
lang-check: fancy ruler

3d-gussner committed:
Define MSG_FS_V_04_OR_NEWER and MSG_FS_V_03_OR_OLDER c=14

3d-gussner committed:
Define MSG_SELFTEST_FS_LEVEL

3d-gussner committed:
Update French translation MSG_RESUME_PRINT
